src/allotropy/parsers/lines_reader.py

```diff
@@ -14,34 +14,36 @@
 EMPTY_STR_PATTERN = r"^\s*$"
 
 
-def read_to_lines(named_file_contents: NamedFileContents) -> list[str]:
+def detect_encoding(bytes_content: bytes) -> str:
```

Comment:
I can't tell what the change in logic is here. If there is a change can you clarify what it is? If not, I don't think there is sufficient justification to refactor the code here.

Reply:
I second this. Plus, if there's a need to refactor the `lines_reader` which is used by several other adapters, it should have its own PR.

Reply:
The idea is to decouple this functions related to read raw files. I can remove this changes from the PR if needed.